doc/man1/flux-jobs.rst

```diff
@@ -356,6 +359,56 @@ include:
    ``{instance.resources.all.ncores}``
 
 
+CONFIGURATION
+=============
+
+The ``flux-jobs`` command supports registration of named output formats
+in configuration files. The command loads configuration files from
+``flux-jobs.EXT`` from the following paths in order of increasing precedence:
+
+ * ``/etc/xdg/flux``
+ * ``$XDG_CONFIG_DIRS/flux``
+ * ``$HOME/.config/flux``
+ * ``XDG_CONFIG_HOME/flux``
```

Comment:
Thanks for finding that!

Yeah, it makes more sense that the defaults are used only when the XDG variables aren't set, not in addition to the XDG variables like I assumed at first (now I'm scratching my head wondering why I thought that in the first place).

> I wonder just using that package makes sense?

It is tempting with Python to just use available libraries add a `requirements.txt` and have users `pip install` away, but I'm unsure if this is good practice for system software. Though in this case the package is already available on TOSS 4, so perhaps pyxdg is more ubiquitous than other non-standard python libraries. Still it is one more requirement to add, and this case seems pretty trivial (literally it would replace a couple lines of code?) so I admit I'm on the fence. I'll use it if you feel strongly about it.

Reply:
> I'll use it if you feel strongly about it.

I do not.

Reply:
Ok, I pushed a fixup commit that I think fix the config directory search path (and also gets the XDG_CONFIG_DIRS in the right order). Let me know what you think and I can squash.

I also added a commit that sets the XDG_CONFIG_* variables to nonexistent directories for the testsuite so that system or user config does not interfere with tests.